content/en/kb/butterfly-network/index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "How to join the Butterfly network."
+description: "A guide for how you can join the Butterfly network"
+date: 2022-03-17T12:00:35+01:00
+lastmod: 2021-11-16T12:00:35+01:00
+draft: false
+menu:
+  kb:
+    parent: "browse"
+toc: false
+pinned: false
+types: ["solution"]
+areas: ["Lotus Node", "Lotus Miner"]
+---
+
+## Problem
```

Comment:
We don't need to follow problem/solution format here. This is not a solution but an article. 
```suggestion
## Butterfly Network
```